src/clj/villagebook/field_value/db.clj

```diff
@@ -0,0 +1,9 @@
+(ns villagebook.field-value.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [villagebook.config :as config]))
+
+(defn add-values!
+  ([value-rows]
+   (add-values! (config/db-spec) value-rows))
+  ([conn value-rows]
+   (jdbc/insert-multi! conn :field_values value-rows)))
```

Comment:
FYI this would do as many db operations as the number of maps in value-rows. Just something to be aware of.
https://clojure.github.io/java.jdbc/#clojure.java.jdbc/insert-multi!